cmd/csidriver/main_test.go

```diff
@@ -59,7 +59,7 @@ func backgroundRun_watchForConfigChanges(t *testing.T, cfgFilePath string, doneC
 // configuration file has changed.
 func TestMain_exitWhenConfigChanges(t *testing.T) {
 	// original configuration file path, using the same than testing configs
-	cfgFilePath := "../test/config/config.yaml"
+	cfgFilePath := "../../test/config/config.yaml"
```

Comment:
not sure why this is necessary ... but the e2e's should sort it out once we have fixed the image building stuff between here and the updated openshift/release PR